### PR TITLE
BAT-7559 - Reinstate EmbeddedListField

### PR DESCRIPTION
**Pull request by asiebert**
:arrow_upper_left: https://work4labs.atlassian.net/browse/BAT-7559 

-----

No obvious reason to drop it, and used in JOP. Seems to work as-is.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 except ImportError:
     pass
 
-VERSION = '0.4.7'
+VERSION = '0.4.6+work4.2'
```
Comment (olivierlefloch) [inline setup.py]: 'work4.2' looks REALLY weird, it looks like version '4.2'. Unavoidable? ✅

**Review by olivierlefloch — APPROVED** [review]

**Comment by olivierlefloch** [comment]
@asiebert beware, unit tests fail. But perhaps they already failed?

File: setup.py (hunk 1 above)
Comment (asiebert) [reply setup.py]: I'm afraid so. Our "slug" is awkward for most short references. :-P

**Comment by asiebert** [comment]
Correct, tests have always been failing on that repo, cf. [latest on master](https://circleci.com/gh/Work4Labs/django-tastypie-mongoengine/8). That said, it's gotten worse in this PR – amending. :construction:

**Comment by asiebert** [comment]
Was missing e9475e8 to fix some Django deprecated data structure. Tests are now failing identically to master... "yay". :white_check_mark:

**Pull request merged**